book_haven_api/tests/requests/libraries/books/test_add_book_to_library.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from django.urls import reverse
+from rest_framework.test import APIClient
+from book_haven_api.models import Library, Book
+
+@pytest.fixture
+def Libraries():
+    library1 =  Library.objects.create(
+        id=1,
+        name='Library1',
+        street='1234 Library1 St.',
+        city='Library1 City',
+        state='CA',
+        zip='12345',
+        lat=123.456,
+        lon=123.456,
+    )
+
+    return library1
+
+
+@pytest.mark.django_db
+def test_add_book_to_library(Libraries):
+    print('add_book_to_library_happy_path')
+    library1 = Libraries
+    url = reverse('get_library_books', kwargs={'library_id': library1.id})
+    client = APIClient()
```

Comment:
As you grow your testing suite, you'll probably want to move this to at least a file level global variable, but even potentially a module or session level (or Pytest fixture). That way you aren't paying a cost to instantiate a new client for every single test.